### ISSUE TITLE
verification reports C2114 Found invalid dependency for "pandas >=0.15"

### ISSUE THREAD

**Issue by parente** [conda/conda-verify#40]
This might be a conda-verify 3.0.1 issue or might be my misunderstanding of what constitutes a valid version spec. 

I have a spec with the following requirements which did not cause a problem with conda-verify-2.0.0.

```
requirements:
  build:
    - python
    - setuptools
  run:
    - python
    - python-hdfs
    - pandas >=0.15
```

With conda-verify 3.0.1, conda-build errors with  C2114 Found invalid dependency "pandas >=0.15" in info/index.json, suggesting the pandas version spec is somehow invalid.

**Comment by mandeep** [comment]
Thanks. I believe this was fixed in 3.0.2.

**Comment by parente** [comment]
Ah, cool. I missed that there was another patch release.
